README.md

```diff
@@ -19,27 +19,29 @@ SPDX-FileCopyrightText: 2024 router <messagebus@vk.com>
 SPDX-FileCopyrightText: 2025 Aiden <28298836+Aidenkrz@users.noreply.github.com>
 SPDX-FileCopyrightText: 2025 Piras314 <p1r4s@proton.me>
 
+Edited for Trauma Station by @Paradoxum on Discord :P
+
 SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
-<p align="center"> <img alt="Space Station 14" width="880" height="300" src="https://github.com/Goob-Station/Goob-Station/blob/master/Resources/Textures/Logo/logo.png" /></p>
+<p align="center"> <img width="1500" height="857" alt="Trauma3" src="https://github.com/user-attachments/assets/77d53a00-ca85-4da8-9ed5-1faf910e9e3d" /></p>
 
 This is a fork from the primary repo for Space Station 14 called Goob Station. To prevent people forking RobustToolbox, a "content" pack is loaded by the client and server. This content pack contains everything needed to play the game on one specific server this is the content pack for Goob Station.
```

Comment:
> This is a fork from the primary repo for Space Station 14 called Goob Station.

Goob isn't necessarily the primary repo, and our relationship with Goob and WizDen is a little more complicated than that at this point. I'd defer it to @deltanedas on how to phrase this.

> This content pack contains everything needed to play the game on one specific server this is the content pack for Goob Station.

Might wanna reword this one and change Goob to Trauma